nanoFirmwareFlasher.Library/Exceptions/CantConnectToJLinkDeviceException.cs

```diff
@@ -14,18 +14,35 @@ namespace nanoFramework.Tools.FirmwareFlasher
     [Serializable]
     public class CantConnectToJLinkDeviceException : Exception
     {
+        /// <summary>
+        /// Cannot connect to the JLINK device exception.
```

Comment:
please use the official naming and case
```suggestion
        /// Cannot connect to the J-Link device exception.
```